scripts/trace_runfolder.py

```diff
@@ -35,6 +36,9 @@ def get_actions_from_executions(executions, headers):
 def filter_actions_by_name(actions, name):
     return (action for action in actions if action["action"]["name"] == name)
 
+def sort_actions_by_timestamp(actions):
+    sort_actions = sorted(actions, key=lambda x:datetime.datetime.strptime(x['start_timestamp'].split(".")[0], "%Y-%m-%dT%H:%M:%S"))
```

Comment:
I'd suggest breaking this into two lines, with e.g. `start_time  = x['start_timestamp'].split(".")[0]` together with a comment on the input date format, since that might now be immediately obvious to the reader.